src/controllers/UserController.ts

```diff
@@ -25,13 +19,13 @@ class AuthController {
 			const response = new ApiResponse(res)
 			const inputData: UserUpdatePayload = req.body
 
-			const listUserData = await User.findOne({userId: inputData.userId})
+			const listUserData = await User.findById({userId: inputData.userId})
 			if (!listUserData) {
 				throw new Error("User not found")
 			}
-
+		
 		// update  
-		const data = await User.updateOne({userId: inputData.userId} , {$set: inputData})
+		const data = await User.findByIdAndUpdate({userId: inputData.userId} , {$set: inputData, $isVerified: false})
```

Comment:
for findOneAndUpdate
we need to send only id as first parameter not whole object